beetsplug/deezer.py

```diff
@@ -54,7 +76,15 @@ def album_for_id(self, album_id):
             return None
 
         album_data = requests.get(self.album_url + deezer_id).json()
-        artist, artist_id = self.get_artist(album_data['contributors'])
+        if 'error' in album_data:
+            self._log.debug(f"Error fetching album {album_id}: "
+                            f"{album_data['error']['message']}")
+            return None
```

Comment:
Cool; this seems like an important addition! Maybe it deserves its own changelog bullet point?

Reply:
Not sure if this is required, but added it anyways :)